plugins/node.d/snmp__rdp_users.in

```diff
@@ -92,15 +92,14 @@ users.label users
 users.draw LINE2
 users.info Number of users logged in.
 EOC
-    exit 0;
+	exit 0;
 }
 
 my($session, $error) = Munin::Plugin::SNMP->session();
 die "Failed to create session: $error" unless $session;
 
 my $process_names = $session->get_entries(-columns => [ $oid ]);
 my $user_count = $process_names ? grep /^rdpclip\.exe$/i, values %$process_names
-                                : 'U';
+																: 'U';
```

Comment:
This is quite a deep level of indentation, or? Do you use two spaces per tab in your editor?

The editor-configurable amount of whitespace makes it impossible to reach universal alignment with tabs. Thus we should probably just increase the level of indentation by one in cases like above?